ci/unreal-engine.version

```diff
@@ -1,2 +1,2 @@
-HEAD 4.25-SpatialOSUnrealGDK
-HEAD 4.24-SpatialOSUnrealGDK
+HEAD 4.25-SpatialOSUnrealGDK
+HEAD 4.24-SpatialOSUnrealGDK
```

Comment:
I don't think it matters but this error symbol here is because of a lack of new line at the end of the file